build/build/src/engine/context.rs

```diff
@@ -199,6 +199,26 @@ impl RunContext {
             };
             sbt.call_arg("syntax-rust-definition/Runtime/managedClasspath").await?;
         }
+        if self.config.build_native_runner {
+            env::ENSO_LAUNCHER.set(&engine::EngineLauncher::DebugNative)?;
+        } else {
+            env::ENSO_LAUNCHER.set(&engine::EngineLauncher::Shell)?;
+        }
+
+        // TODO: Once the native image is production ready, we should switch to
+        // EngineLauncher::Native on release versions.
+        // See tasks in https://github.com/orgs/enso-org/discussions/10121
+        /*let version = versions_from_env()?.unwrap();
+        let is_release = version.release_mode;
+        let kind = Kind::deduce(&version)?;
+        if is_release {
+            env::ENSO_LAUNCHER.set(&engine::EngineLauncher::Native)?;
+        } else if kind == Kind::Dev {
+            env::ENSO_LAUNCHER.set(&engine::EngineLauncher::Shell)?;
+        } else {
+            env::ENSO_LAUNCHER.set(&engine::EngineLauncher::default())?;
```

Comment:
If we leave `ENSO_LAUNCHER` unset, then we don't need this for enabling _native image_ in production at all. I think.

Reply:
Simplified in 318c786a79d4f8b220180b45be11e278e3fa8c88